src/js/common/schemaform/reducers/index.js

```diff
@@ -76,7 +80,14 @@ export default function createSchemaFormReducer(formConfig) {
         id: false,
         timestamp: false,
         hasAttemptedSubmit: false
-      }
+      },
+      savedStatus: 'not-attempted',
+      loadedStatus: 'not-attempted',
+      version: formConfig.version,
+      formId: formConfig.formId,
+      disableSave: formConfig.disableSave,
+      // loadedData: undefined
+      migrations: formConfig.migrations
```

Comment:
I'm not convinced this is the best place to store the migrations, but I wasn't sure how else to get it from the form config to the loading function.